src/app/gameLogic.js

```diff
@@ -0,0 +1,12 @@
+function testForCollision(object1, object2)
+    {
+        const bounds1 = object1.getBounds();
+        const bounds2 = object2.getBounds();
+
+        return (
+            bounds1.x < bounds2.x + bounds2.width
+            && bounds1.x + bounds1.width > bounds2.x
+            && bounds1.y < bounds2.y + bounds2.height
+            && bounds1.y + bounds1.height > bounds2.y
+        );
+    }
```

Comment:
Looks good, do you know if there are any prebuild functions in the library that can do this as well?